hvac/v1/__init__.py

```diff
@@ -21,6 +21,9 @@
 
 
 class Client(object):
+    _version = 1
```

Comment:
Shouldn't this be a instance attribute rather than a class attribute? As-is, if someone wanted to instantiate two Client classes, one pointed to a Vault instance with kv version 1 and one with kv version 2, they wouldn't be able to. I could also imagine a case where someone wants to use the client with a kv version 1 secret backend under the path `secrets/` and a version 2 backend under `kv/`. In that case, we wouldn't want this to be an attribute at all. I.e., perhaps it makes more sense to detect the version every time we need to send any request to the kv backend? If we went that route, it would make sense to add a toggle for kv version auto-detection in case folks don't want the client to preform the extra requests needed to determine the version on each kv method call.